harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/geonet/Aligner.java

```diff
@@ -166,6 +166,7 @@ public HarvestResult align(Set<RecordInfo> records, List<HarvestError> errors) t
 		//--- remove old metadata
 
 		for (String uuid : localUuids.getUUIDs()) {
+		    //FIXME can we do it on one step?
```

Comment:
This comment isn't clear to me.   We need to check if the process has been cancelled every iteration so what do you mean by doing it in one step?